static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -32,7 +30,6 @@ define([
     reduce
 ) {
     var urlBuilders = {
-        bestbuy:        defaultUrlBuilder('money/bestbuys'),
```

Comment:
There will presumably be a creative template in DFP that uses this endpoint, which should be removed or disabled.